<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagerDlg</class>
 <widget class="QDialog" name="FileManagerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTreeView" name="treeView">
   <property name="geometry">
    <rect>
     <x>649</x>
     <y>109</y>
     <width>600</width>
     <height>651</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>68</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="icon">
    <iconset resource="fastvideo.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="shareBtn">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>68</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>共享导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="diskBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>68</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>磁盘导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ftpBtn">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>68</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>FTP导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileplayBtn">
   <property name="geometry">
    <rect>
     <x>1074</x>
     <y>68</y>
     <width>81</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
   <property name="icon">
    <iconset resource="fastvideo.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="sourcePath_label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>55</y>
     <width>761</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>e:/</string>
   </property>
  </widget>
  <widget class="SourcefileWidget" name="sourcefileWidget" native="true">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>90</y>
     <width>1278</width>
     <height>681</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SourcefileWidget</class>
   <extends>QWidget</extends>
   <header>sourcefilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
